### DISCUSSION TITLE
[Feature] Account switching after x dm sent

### DISCUSSION THREAD

**Discussion by Alex9432** [hbollon/IGopher#23] [General]
Hello
Is there a feature for the dm bot to change account after a certain number of dm sent ?
Thanks

**Comment by hbollon** [comment]
Hi!
Sorry for the late reply, I completely missed your message.
Actually, IGopher doesn't support this feature, however, it seems possible to do that by following IGopher's development direction.
I invite you to open a feature request issue if you would like the implementation of this feature :+1:
